Sources/Views/ALKChatBar.swift

```diff
@@ -400,6 +399,7 @@ open class ALKChatBar: UIView, Localizable {
         self.configuration = configuration
         isMicButtonHidden = configuration.hideAudioOptionInChatBar
         initializeView()
+        lineView.backgroundColor = configuration.aboveChatbarLineViewColor
```

Comment:
you can change the config name to chatbartoplineviewcolor for something similar to this

Reply:
okay